### ISSUE TITLE
inference speed

### ISSUE THREAD

**Issue by wynterr** [KaihuaTang/Scene-Graph-Benchmark.pytorch#26]
Hi, I'm just testing the pretrained faster_rcnn you provided and I found the speed is really slow. It took 0.3s to inference on average, and around 3 hours for the entire test set on 1 gpu. I wonder what is it normal speed for inference?
BTW, I met some trouble installing apex and ended up with installing the python only version of it. Does it have something to do with the slow speed?
Thanks!

**Comment by KaihuaTang** [comment]
I think it's the normal speed, but if you found something we can optimize, pls let me know. APEX is used for distributed training/testing. It won't affect the inference speed for each image, but if you got multiple GPUs, the overall time cost in inference will definitely be reduced significantly.

**Comment by wynterr** [comment]
> 
> 
> I think it's the normal speed, but if you found something we can optimize, pls let me know. APEX is used for distributed training/testing. It won't affect the inference speed for each image, but if you got multiple GPUs, the overall time cost in inference will definitely be reduced significantly.

OK，Thanks！
